.github/ISSUE_TEMPLATE/feature_request.yaml

```diff
@@ -0,0 +1,52 @@
+name: Feature request
```

Comment:
Title of [other templates](https://github.com/euphony-io/euphony/issues/new/choose) has cute emoji 😄 
So, how about adding emoji in name?

Reply:
🚀 I recommend `rocket`!!

Reply:
```rocket``` is already used at sample app proposal 👀 is it okay??

Reply:
Sorry kk

Reply:
I think that 👀 is good!!

Reply:
how about 📔 ?!

Reply:
I think that 📔 is good too 😆

Reply:
I added emoji !!